framework/caching/CCache.php

```diff
@@ -105,7 +105,9 @@ public function get($id)
 		if($value===false || $this->serializer===false)
 			return $value;
 		if($this->serializer===null)
-			$value=unserialize($value);
+            $value=false;
+            if($value!==null)
```

Comment:
This condition does not make sense.

Reply:
fixed - that was an accidental copy/paste :(